cmd/validator/flags/flags.go

```diff
@@ -322,6 +323,26 @@ var (
 		Usage: "Enables more verbose logging for counting down to duty",
 		Value: false,
 	}
+
+	// ValidatorsProposerConfigDirFlag defines the path or URL to a file with proposer config.
+	ValidatorsProposerConfigDirFlag = &cli.StringFlag{
```

Comment:
This is a file path and not a dir path so we should rename the flag and clarify the usage to say it is a path to a JSON file containing validator mappings to ETH addresses for receiving transaction fees when proposing blocks